packages/features/calendars/DestinationCalendarSelector.tsx

```diff
@@ -117,10 +118,6 @@ const DestinationCalendarSelector = ({
         })),
     })) ?? [];
 
-  // Get primary calendar, which is shown in the placeholder since this is the calendar that will
-  // be used when no destination calendar is selected.
-  const primaryCalendar = query.data.destinationCalendarEmail;
-
```

Comment:
I wouldn't say I liked this, as this wouldn't be the actual destination calendar.